pydantic_ai_slim/pydantic_ai/agent.py

```diff
@@ -413,6 +419,11 @@ def __init__(
         self._system_prompt_dynamic_functions = {}
 
         self._max_result_retries = output_retries if output_retries is not None else retries
+        self._builtin_tools: list[AbstractBuiltinTool] = []
+
+        for tool in builtin_tools:
+            self._builtin_tools.append(tool)
```

Comment:
This can just be `self._builtin_tools = builtin_tools` now that we've dropped the string option, right?